### DISCUSSION TITLE
Too many statues/fountains

### DISCUSSION THREAD

**Discussion by legitalk** [OpenTTD/OpenTTD#10221] [Features]
Version of OpenTTD
OpenTTD nightly (20221205)

I know this problem has been around for over 20 years but is there really nothing to fix this?

I've been playing classic scenario since 2000, I'm currently in 2823, and every city (all 100K+ population) have a lot of statues/fountains that I need to destroy every year because it affects the game (number of passengers at stations).

Is there a script that will do this automatically or how can it be improved? I don't believe it's not possible to just set up say 1 statue/fountain per 10K population.

Picture:
![DURSoft Computers, 2822-07-24](https://user-images.githubusercontent.com/6940865/205920646-dbc6e79f-6bd9-4329-9ef0-3e4899b66833.png)

**Comment by LC-Zorg** [comment]
I don't think the frequency of monuments is a particular problem. I have seen their accumulation in one place more than once, which I admit didn't look very good, but what you show is rather an extreme and not very common case, in addition significantly enhanced by the original road layout used. If you use a 3x3 grid layout these monuments will be barely noticeable.

A very typical city view with a 3x3 grid
![Rare monuments](https://user-images.githubusercontent.com/67662046/206857883-dc533235-5a80-4548-9a02-350bdcb1a4b5.png)
Are there too many statues?

Besides, even quite a large number of these monuments and fountains in the city I don't consider a disadvantage, but rather an advantage, because they slightly dilute the already very dense arrangement of buildings, thus improving the structure of the city, making it less monolithic. If I were to consider anything a problem with monuments, it's not their number, but their arrangement and repetition. So perhaps it would be useful to add a few different versions of these monuments, or add a new type of building, such as parks, markets or squares (all 1x1), which would alternate with monuments and fountains. Perhaps it would also be worth adding conditioning so that the same monuments do not appear next to each other. It's not uncommon to have a number of fountains next to each other (e.g. there's a fountain alley in my city), though it would be helpful to add other possibilities so that it doesn't happen too often.

However, if you consider the low number of passengers at the stations to be the main problem, the player still has a few ways to fix it himself, without any change in the game code:
1. He can change the street grid, because with the one you are using, 2/3 of the city's buildings are streets. If you use a 3x3 grid, a city of the same size will have almost twice as many buildings and therefore twice as many passengers.
2. He can turn on the original passenger generation model - then he will definitely not complain about the lack of them. ;)
3. He may add one of the available building sets (NewGRF) to the game. There are a few of them. The best one I can recommend right now is Swedish Houses Set. Apart from it, there are: Japan Buildings, Korean Style Buildings, Improved Town Layout, Polish Building Set, Total Town Replacement Set and maybe some other. In addition to the add-ons available in the game, you can also find Canadian City Buildings and OpenGFX Mars Habitats. At least three other sets are in development.
4. As a last resort, he can also create his own set of buildings with his own placement rules. It's probably even possible to create a simple add-on that just disables the ability to build statues and fountains.
